extension-manifest-v3/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@ghostery/extension-manifest-v3",
   "private": true,
-  "version": "9.5.0",
+  "version": "9.6.0",
```

Comment:
9.5.0 is already upcoming version. It was bumped with onboarding, but not yet released.

Reply:
fixed